ui/app/AppLayouts/Wallet/popups/AddCustomTokenModal.qml

```diff
@@ -57,7 +57,7 @@ ModalPopup {
     }
 
     property var getTokenDetails: Backpressure.debounce(popup, 500, function (tokenAddress){
-        RootStore.customTokenList.getTokenDetails(tokenAddress)
+        RootStore.walletTokensModule.getTokenDetails(tokenAddress)
```

Comment:
I might be wrong but I'm not sure if we want to explose entire modules on the store..

Reply:
Nevermind, seeing that this is just a sub section. Maybe we should call this differently then?

Just thinking out loud.